pkg/apis/flowcontrol/types.go

```diff
@@ -296,6 +298,17 @@ type FlowSchemaStatus struct {
 	// +listMapKey=type
 	// +optional
 	Conditions []FlowSchemaCondition
+
+	// `controllerSemVer` holds the semantic version of the controller
+	// that set the status.  A controller of lesser semantic version
+	// is not allowed to subsequently update the status.
+	ControllerSemVer string
+
+	// `evaluatedPriorityLevelConfigurationReference` reports what the
+	// kube-apiservers think of the Spec's reference to a priority
```

Comment:
how does this interact with multiple kube-apiservers?

Reply:
They should all reach the same answer.
It's conceivable that if they are of different releases they might have different answers.  That's why the `ControllerSemVer` is there.